.well-known/security.txt

```diff
@@ -0,0 +1,10 @@
+# The SRCF is a volunteer-run student society of the University of
+# Cambridge that operates a variety of shared multi-user systems. Please
+# use the contact details below to report security issues such as hacked
+# accounts, XSS vulnerabilities and SQL injection vulnerabilities, among
+# others.
+#
+Contact: mailto:soc-srcf-admin@lists.cam.ac.uk
```

Comment:
In the interest of committing to an address in the long term, let's make this `sysadmins@srcf.net`, just in case lists.cam.ac.uk goes away in the meantime.